File: AndroidApp/IMSApp/src/mobile/IMS/ReportData.java
package mobile.IMS;

import java.io.Serializable;
import java.util.Date;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Namespace;
import org.simpleframework.xml.Order;
import org.simpleframework.xml.Path;
import org.simpleframework.xml.Root;

@Root(name = "report")
@Namespace(reference="http://schemas.datacontract.org/2004/07/IMS.Api")
@Order(elements={"id","operator-id", "description", "priority", "location", "datetime"})
public class ReportData implements Serializable {
	private static final long serialVersionUID = 134836923434623428L;
	@Element(name = "id", required = true)
	private String uuid;

	@Element(name = "operator-id", required = true)
	private String operatorID;

	@Element(name = "description", required = false)
	private String textDescription;

	@Element(name = "priority", required = true)
	private String priority;

	@Element(name = "location", required = false)
	private String location;

	@Element(name = "datetime", required = true)
	private String dateTime;

	public ReportData() {
	}

	public String getUuid() {
		return uuid;
	}

	public void setUuid(String uuid) {
		this.uuid = uuid;
	}

	public String getOperatorID() {
		return operatorID;
	}

	public void setOperatorID(String operatorID) {
		this.operatorID = operatorID;
	}

	public String getTextDescription() {
		return textDescription;
	}

	public void setTextDescription(String textDescription) {
		this.textDescription = textDescription;
	}

	public String getPriority() {
		return priority;
	}

	public void setPriority(String priority) {
		this.priority = priority;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

	public String getLocation() {
		return this.location;
	}

	public void setLocation(String location) {
		this.location = location;
	}

	public void setDateTime(String dateTime) {
		this.dateTime = dateTime;
	}

	public void setDateTime(Date date) {
		this.dateTime = date.toGMTString();
	}
	
	public String getDateTimm() {
		return this.dateTime;
	}

}

File: IMS.Api/ReportData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace IMS.Api
{
	//    @Element(name = "id")
	//private String uuid;

	//@Element(name = "operator-name")
	//private String operatorName;
	
	//@Element(name="operator-id")
	//private String operatorID;

	//@Element(name = "description")
	//private String textDescription;

	//@Element(name = "has-photo")
	//private boolean hasPhoto;



	[DataContract(Name = "report")]
	public class ReportData
	{
		[DataMember(Name = "id", IsRequired = true, EmitDefaultValue = false, Order = 0)]
		public Guid ReportID { get; set; }

		[DataMember(Name = "operator-id", IsRequired = true, Order = 1)]
		public Guid OperatorID { get; set; }

		[DataMember(Name = "description", Order = 2)]
		public string Description { get; set; }

		[DataMember(Name = "priority", Order = 3)]
		public string Priority { get; set; }

		[DataMember(Name = "location", Order = 4)]
		public string Location { get; set; }

		[DataMember(Name = "datetime", Order = 5)]
		public String DateTime { get; set; }

	}
}
